api/envoy/config/core/v3/protocol.proto

```diff
@@ -85,7 +85,7 @@ message QuicProtocolOptions {
       [(validate.rules).uint32 = {lte: 25165824 gte: 1}];
 
   // The number of timeouts that can occur before port migration is triggered for QUIC clients.
-  // This defaults to 1. If set to 0, port migration will not occur on path degrading.
+  // This defaults to 4. If set to 0, port migration will not occur on path degrading.
```

Comment:
The change LGTM. From API perspective the server is expected to explicitly set the values of WKTs.
That said, I wonder whether it's a user-observable change. If so,
1) please add a release note.
2) consider making this change runtime guarded.